apps/landingPage/src/components/Info/Features.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+import React from 'react';
+import { Heading } from '@kiwicom/orbit-components/lib/';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import styled from 'styled-components';
+
+import { BREAKPOINTS } from '../../mediaQueriesConfig';
+
+const features = ['Search', 'Booking', 'MMB', 'Payments'];
+
+const renderCards = (features: Array<string>) => {
```

Comment:
could you use `$ReadOnlyArray<string>` instead?

Reply:
Sure! changed now